### DISCUSSION TITLE
<title>Can't get into wp admin due to fatal error. Tried everything.

### DISCUSSION THREAD

**Discussion by CEOGPS** [community/community#43334] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Updated wp and plugins now getting "Fatal error: Cannot declare class WP_Importer, because the name is already in use in /home/customer/www/treewurk.com/public_html/wp-admin/includes/class-wp-importer.php on line 6". Tried everything and nothing is working. Tried live chat but was on for two hours waiting on an answer. Can anyone help?

**Comment by emilburganov** [comment]
https://wordpress.stackexchange.com/questions/359257/fatal-error-please-help-me
